app/controllers/publishers/vacancies/application_controller.rb

```diff
@@ -1,7 +1,32 @@
 require "indexing"
 
 class Publishers::Vacancies::ApplicationController < Publishers::BaseController
-  include Publishers::JobCreationHelper
+  def steps_config
+    @steps_config = {
+      job_location: { number: 1, title: I18n.t("jobs.job_location") },
+      schools: { number: 1, title: I18n.t("jobs.job_location") },
+      job_details: { number: 2, title: I18n.t("jobs.job_details") },
+      pay_package: { number: 3, title: I18n.t("jobs.pay_package") },
+      important_dates: { number: 4, title: I18n.t("jobs.important_dates") },
+      supporting_documents: { number: 5, title: I18n.t("jobs.supporting_documents") },
+      documents: { number: 5, title: I18n.t("jobs.supporting_documents") },
+      applying_for_the_job: { number: 6, title: I18n.t("jobs.applying_for_the_job") },
+      job_summary: { number: 7, title: I18n.t("jobs.job_summary") },
+      review: { number: 8, title: I18n.t("jobs.review_heading") },
+    }.freeze
+  end
+
+  def steps_adjust
+    current_publisher_is_part_of_school_group? ? 0 : 1
+  end
+
+  def step_current
+    @step_current = defined?(step) ? step : :job_location
```

Comment:
I don't see how it's possible for `defined?(step)` to be true, as it's not been defined within this method.

Reply:
Ah: I can see `def step; :documents; end` in the documents controller. Ignore me.